spring-cloud-gcp-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -29,6 +29,12 @@
       "type": "java.lang.Boolean",
       "description": "Auto-configure Google Cloud Storage components.",
       "defaultValue": true
+    },
+    {
+      "name": "spring.cloud.gcp.vision.enabled",
```

Comment:
Why this property can't go to the `CloudVisionProperties` ?

Reply:
Have you missed this one?

Thanks

Reply:
You could, but putting it there won't do anything because `spring.cloud.gcp.vision.enabled` is used within a `@ConditionalOnProperty` and these [are processed before `ConfigurationProperties` are parsed](https://github.com/spring-projects/spring-boot/issues/2282). Any default value set in CloudVisionProperties will be ignored. That is why all the `.enabled` properties have been put in here.